addons/mail/static/src/discuss/core/web/discuss_core_web_service.js

```diff
@@ -87,12 +87,12 @@ export class DiscussCoreWeb {
                     if (data.foldStateCount) {
                         thread.foldStateCount = data.foldStateCount;
                     }
-                    if (data.serverFoldState !== thread.state) {
+                    const chatWindow = this.store.chatWindows.find(
+                        (chatWindow) => chatWindow.threadLocalId === thread.localId
+                    );
+                    if (data.serverFoldState !== thread.state || chatWindow?.hidden) {
```

Comment:
I don't understand the diff here, hidden should be irrelevant to sync sate from server